include/openthread/link.h

```diff
@@ -788,8 +788,13 @@ void otLinkFilterClearAddresses(otInstance *aInstance);
 otError otLinkFilterGetNextAddress(otInstance *aInstance, otMacFilterIterator *aIterator, otMacFilterEntry *aEntry);
 
 /**
- * Adds a fixed received signal strength (in dBm) entry for the messages from a given Extended Address in
- * MAC Filter.
+ * Adds the specified Extended Address to the `RssIn` list (or modifies an existing
```

Comment:
The changes here appear to all be from the other PR
https://github.com/openthread/openthread/pull/9442

Reply:
yes thanks! fixed

Reply:
thx, fixed now